test/65-assert-10-assert_symlink_to.bats

```diff
@@ -29,3 +30,12 @@ teardown () {
   [ "${lines[1]}" == "path : $link" ]
   [ "${lines[2]}" == '--' ]
 }
+@test 'assert_symlink_to() <temp_file> <link>: returns 0 if <link> exists and is a symbolic link to <temp_file>' {
+  touch ${TEMP_FOLDER}/file ${TEMP_FOLDER}/notasymlink
+  ln -s ${TEMP_FOLDER}/file ${TEMP_FOLDER}/symlink
+  local -r file="${TEMP_FOLDER}/file"
+  local -r link="${TEMP_FOLDER}/symlink"
+  run assert_symlink_to $file $link
```

Comment:
Please quote the variable expansions.

Reply:
I quoted variable expansions in the functions affected by this PR, but the same comment applies: this issue occurs in other places too.